subsys/bluetooth/services/dis.c

```diff
@@ -154,7 +314,8 @@ static int dis_set(const char *name, size_t len_rd,
 		len = read_cb(store, &dis_manuf, sizeof(dis_manuf) - 1);
 		if (len < 0) {
 			LOG_ERR("Failed to read manufacturer from storage"
-				       " (err %zd)", len);
+				" (err %zd)",
+				len);
```

Comment:
Why did you put `len` on a separate line? Seems to me like you should merge the format string onto just one line - does it really go beyond 100 chars if you do that? If you need to do some splitting (due to hitting line length limits), it'd be better to keep the format string intact while putting the parameters on the next line. Generally, splitting logging string literals across multiple lines is bad because it makes it more difficult to use grep for finding which line in the code cause a specific log output.

Reply:
I think that was the way it was originally. But we could just put everything on one line